core/server/utils/markdown-converter.js

```diff
@@ -0,0 +1,26 @@
+var MarkdownIt  = require('markdown-it'),
+    converter = new MarkdownIt({
+        html: true,
+        breaks: true,
+        linkify: true
+    })
+    .use(require('markdown-it-footnote'))
+    .use(require('markdown-it-lazy-headers'))
+    .use(require('markdown-it-mark'))
+    .use(require('markdown-it-named-headers'), {
+        // match legacy Showdown IDs otherwise default is github style dasherized
```

Comment:
`markdown-it-named-headers` adds IDs to header elements so `# my header` becomes `<h1 id="my-header">my header</h1>`

The dasherized `my-header` is what GitHub Flavoured Markdown (markdown-it, current showdown, etc) outputs by default but our old Showdown renderer didn't dasherize and would instead output `id="myheader"`.

Rather than add a breaking change that would affect links to migrated content I thought it would be better to replicate the old behaviour.